prometheus/alerts/alert.rules

```diff
@@ -226,6 +226,102 @@ groups:
       annotations:
         description: 'Vortex Front End is unreachable. Please follow oncall sop to debug.'
 
+    - alert: IndexerOrdersUnreachable
```

Comment:
could we wild card these alerts and just have a single alert rather than redefining separate alerts for different params. i.e.
`heatseeker_client_indexer_status{endpoint="https://offchain-v2.sei-chain-incentivized.com/*", status_code!="200"}`

Reply:
I was thinking of doing this (effectively in the `runOffchainIndexerCheck`, we would check every endpoint of the offchain indexer, and if any doesn't respond we would alert). But I thought since each alert right now effectively deals with a separate db + separate handler in the dex indexer service we would want the alerts to be specific?